api/models.py

```diff
@@ -82,8 +82,8 @@ class Item(models.Model):
         (DAMAGED, "Damaged")
     )
 
-    item_code = models.CharField(max_length=50, blank=True)
-    serial_number = models.CharField(max_length=50, blank=True)
+    item_code = models.CharField(unique=True, max_length=50, blank=True)
```

Comment:
Having both `unique=True` and `blank=True` as options for the same field will cause an `IntegrityError: duplicate key value violates unique constraint` error if `item_code` and `serial_number` is not provided. Consider doing away with `blank=True` if we're making these fields unique.
This applies for both item_code & serial_number.